src/libraries/System.Net.Quic/src/System/Net/Quic/QuicListener.cs

```diff
@@ -209,13 +209,15 @@ public async ValueTask<QuicConnection> AcceptConnectionAsync(CancellationToken c
     /// <param name="clientHello">The TLS ClientHello data.</param>
     private async void StartConnectionHandshake(QuicConnection connection, SslClientHelloInfo clientHello)
     {
+        bool wrapException = true;
```

Comment:
we could if we feel `CancellationTokenSource.CreateLinkedTokenSource` can fail and throw. Otherwise I don't really see place where it would matter.

Reply:
It can fire the cancellation synchronously if the token gets or already is canceled. 
Even if that is unlikely in our case, I'd still prefer we follow the same pattern as you did with SSL _validationCallback. It's one assignment, it's not gonna hurt any perf.

Reply:
make sense. updated.